OmiseSDK/CardNameTextField.swift

```diff
@@ -7,7 +7,7 @@ import UIKit
 public class CardNameTextField: OmiseTextField {
     /// Boolean indicating wether current input is valid or not.
     public override var isValid: Bool {
-        return !text.isNilOrEmpty
+        return !text.isNilOrEmpty && (text ?? "").rangeOfCharacter(from: .decimalDigits) == nil
```

Comment:
I think we still should allow the decimal string in the name. We should not restrict too much for the input. If the problem is that a user put the credit card in the name field then, how's about checking if the text is not a valid credit card number instead?

Also I think `(text ?? "")` would affect the build time and it makes the code a little. bit harder to read. We should adopt the guard statement here

Reply:
That would be good if we check the text is not a valid credit card number on the field. 
But why cardholder name field still need to allow the decimal string? I'm not sure can cardholder names contain numbers.

Reply:
Since we are not sure so we should not disallow that. I think there may be a case when someone have something like `3rd` in the name

Reply:
Yeh. I found some source about that too. https://www.quora.com/Can-human-names-contain-numbers 👍

Reply:
@pitiphong-p After I'm checking with our API. We didn't allow the decimal string on the name.
If user insert card name that contains decimal string, our API will return `invalid card`. 😨

Reply:
Could we follow our API first?

Reply:
Well, I think it's OK. We can allow it in case we allow a decimal in the API in the future